app/themes/addiction/css/style.css

```diff
@@ -343,6 +343,26 @@ button {
   left:-20px;
   opacity:0;
 }
+.inner_slider_div {
```

Comment:
We are currently not maintaining `app/themes/addiction/css/style.css` file. Please refer to css section in `HOW TO GUIDE.md`  and put the css rules accordingly,